src/services/index.js

```diff
@@ -2,4 +2,5 @@ export { default as Auth } from "./Auth"
 export { default as HealthInstitutionService } from "./HealthInstitutionService"
 export { default as RoleService } from "./RoleService"
 export { default as TechnologyService } from "./TechnologyService"
-export { default as ActivityService } from "./ActivityService"
\ No newline at end of file
+export { default as ActivityService } from "./ActivityService"
+export { default as ExecutionService } from "./ExecutionService"
```

Comment:
Só adicionar uma nova linha no fim do arquivo

```suggestion
export { default as ExecutionService } from "./ExecutionService"
```

Reply:
Se quiser, pode deixar que o `npm run format` resolve :D